docs/app/Examples/elements/Input/Variations/InputExampleRightLabeled.js

```diff
@@ -7,12 +7,12 @@ const options = [
   { text: '.org', value: '.org' },
 ]
 
-const InputRightLabeled = () => (
+const InputExampleRightLabeled = () => (
   <Input
-    label={<Dropdown defaultValue='.com' options={options} />}
+    label={<Dropdown selection defaultValue='.com' options={options} />}
```

Comment:
The `selection` prop should also not be added here:

### with `selection`
![image](https://cloud.githubusercontent.com/assets/5067638/19407800/80dc48d8-9261-11e6-9010-38b0e051b1a4.png)

### without `selection`

![image](https://cloud.githubusercontent.com/assets/5067638/19407797/693df0d2-9261-11e6-8649-8dfbfc944324.png)